src/StreamrClient.ts

```diff
@@ -350,6 +350,10 @@ export class StreamrClient extends EventEmitter { // eslint-disable-line no-rede
         return this.publisher.rotateGroupKey(...args)
     }
 
+    rekey(...args: Todo) {
```

Comment:
If this is an internal method, annotate with `/** @internal */`? 

The method is called in `Encryption.test.js`, after stream permission is revoked. Should the revoking trigger the rekeying automatically, without user calling this method?

Reply:
not internal.

> Should the revoking trigger the rekeying automatically, without user calling this method?

Probably, IIRC it did do that in the previous implementation.

Reply:
If public, could update the type definition: `rekey(streamId: string)`

Reply:
Done in e13e8a1a